.gitignore

```diff
@@ -59,6 +59,5 @@ override.tf.json
 .terraformrc
 terraform.rc
 
-open-notebook
```

Comment:
what's the reason for removing this from .gitignore? I would assume it should stay ignored?

Reply:
we have a directory at `kubectl/open-notebook` and git git got unhappy when I changed files in that path and added them. Now that our docs are explicit about moving the binary, I think removing this is okay.

An alternative: can we name either of the directory or binary differently?